plugins/history/chathistoryhandler.h

```diff
@@ -21,6 +25,14 @@ class ChatHistoryHandler : public Kopete::Plugin
 
 	~ChatHistoryHandler();
 	static ChatHistoryHandler *instance();
+
```

Comment:
Since this is a singleton, this is probably going to be allocated in the heap memory. I am not expert in de-allocation of memory, but 

1. Do you think we need to handle de allocation ? 
2. There is something called ` K_GLOBAL_STATIC` that can be used for singleton 
http://api.kde.org/4.0-api/apidox-kde-4.0/kdelibs-apidocs/kdecore/html/group__KDEMacros.html

3. We can use any kind of smart pointers that would destroy itself when its done. 

Give it a thought and let me know what do you think.